library/src/main/java/com/chuckerteam/chucker/internal/ui/transaction/TransactionAdapter.java

```diff
@@ -86,26 +86,33 @@ public void setData(List<HttpTransactionTuple> httpTransactions) {
         final View view;
         final TextView code;
         final TextView path;
+        final TextView operationName;
         final TextView host;
         final TextView start;
         final TextView duration;
         final TextView size;
+        final ImageView graphql;
         final ImageView ssl;
 
         ViewHolder(View view) {
             super(view);
             this.view = view;
             code = view.findViewById(R.id.chucker_code);
             path = view.findViewById(R.id.chucker_path);
+            operationName = view.findViewById(R.id.chucker_operation_name);
             host = view.findViewById(R.id.chucker_host);
             start = view.findViewById(R.id.chucker_time_start);
             duration = view.findViewById(R.id.chucker_duration);
             size = view.findViewById(R.id.chucker_size);
+            graphql = view.findViewById(R.id.chucker_graphql);
             ssl = view.findViewById(R.id.chucker_ssl);
         }
 
         void bind(final HttpTransactionTuple transaction) {
             path.setText(String.format("%s %s", transaction.getMethod(), transaction.getPath()));
+            operationName.setText(transaction.getOperationName());
+            operationName.setVisibility(transaction.isGraphQL() ? View.VISIBLE : View.GONE);
+            graphql.setVisibility(transaction.isGraphQL() ? View.VISIBLE : View.GONE);
```

Comment:
All those lines are about GraphQL, can do something to group them ?

```kotlin
if (transaction.isGraphQL()) {
    operationName.setText(transaction.getOperationName());
    operationName.setVisibility(View.VISIBLE);
    graphql.setVisibility(View.VISIBLE);
}
```

And set them to View.GONE by default.